api/src/main/java/org/apache/iceberg/util/DateTimeUtil.java

```diff
@@ -133,4 +134,56 @@ public static long isoTimestampToMicros(String timestampString) {
     return microsFromTimestamp(
         LocalDateTime.parse(timestampString, DateTimeFormatter.ISO_LOCAL_DATE_TIME));
   }
+
+  public static int daysToYears(int days) {
```

Comment:
I had to move this utility to `api` as transforms are part of `api`. I feel that's okay since `BucketUtil` and a few others are already defined there.

Reply:
should we also move `TestDateTimeUtil` to `api`?

Reply:
Good point, will do.

Reply:
Moved.